custom_news_feed/custom_news_feed.py

```diff
@@ -295,13 +304,34 @@ def addLinks(self, links):
         self.dockwidget.linksScrollArea.setWidgetResizable(True)
         self.dockwidget.linksScrollArea.setWidget(self.dockwidget.widget)
 
+    def checkPublishingDate(self, startdate, enddate):
+
+        ret = True
+        now = datetime.now().isoformat()
+
+        if (startdate is not None) and (enddate is not None):
+            if startdate <= now and enddate >= now: ret = True
+            else: ret = False
+        if (startdate is not None) and (enddate is None):        
+            if startdate > now: ret = False
+        if (enddate is not None) and (startdate is None): 
+            if enddate < now: ret = False
+
+        return ret
```

Comment:
Das ganze kann vereinfacht werden, zb so (ohne gewähr ;))
```suggestion
   def checkPublishingDate(self, startdate, enddate):
    now = datetime.now().isoformat()

    if startdate and enddate:
        return startdate <= now <= enddate
    elif startdate:
        return startdate <= now
    elif enddate:
        return now <= enddate

    return True

```